migtests/scripts/live-migration-run-test.sh

```diff
@@ -137,7 +140,7 @@ main() {
 	yb-voyager initiate cutover to target --export-dir ${EXPORT_DIR} --prepare-for-fall-back false --yes
 
 	for ((i = 0; i < 5; i++)); do
-    if [ "$(yb-voyager cutover status --export-dir "${EXPORT_DIR}" | grep -oP 'cutover to target status: \K\S+')" != "COMPLETED" ]; then
+    if [ "$(yb-voyager cutover status --export-dir "${EXPORT_DIR}" | cut -d ':'  -f 2 | tr -d '[:blank:]')" != "COMPLETED" ]; then
```

Comment:
again, mac doesn't support `grep -P`.  It'd be nice to have these test run locally. speeds up dev!

Reply:
Yeah, makes sense. Much needed!